platform-sdk/swirlds-merkle/src/test/java/com/swirlds/virtual/merkle/reconnect/VirtualMapReconnectTest.java

```diff
@@ -59,9 +60,9 @@ class VirtualMapReconnectTest extends VirtualMapReconnectTestBase {
     @BeforeAll
     static void beforeAll() throws Exception {
         final Configuration config = new TestConfigBuilder()
-                .withValue("merkleDb.keySetBloomFilterSizeInBytes", 2 * MEBIBYTES_TO_BYTES * BYTES_TO_BITS)
-                .withValue("merkleDb.keySetHalfDiskHashMapSize", "10000")
-                .withValue("merkleDb.keySetHalfDiskHashMapBuffer", "1000")
+                .withValue(MerkleDbConfig_.KEY_SET_BLOOM_FILTER_SIZE_IN_BYTES, 2 * MEBIBYTES_TO_BYTES * BYTES_TO_BITS)
```

Comment:
just curious - why do these config classes have an underscore after their names? Is this some sort of a temporary name to be changed later?

Reply:
@imalygin the classes are generated at build time. By doing so you do not need to type the string by hand. Since this is similar to the JPA meta model (https://www.baeldung.com/hibernate-criteria-queries-metamodel#static-metamodel) I decided to use the same pattern (`_` as suffix for the generated classes).